lambda-s3/README.md

```diff
@@ -0,0 +1,12 @@
+# image-lambda
+image-lambda
+
+# How-to
+- Log-in to AWS console and upload a file to the resizer1000 bucket. A new, resized file will then be created i nthe resizer1000-resized bucket.
+
+# Issues
+- I ran into issues with conflicting regions; that certain module were set to US-west and others to US-east. Additionally, the zip file had to be created correctly  in order to work with the function.
+
+# Images
+- [Lambda1](../Lambda1.png)
+- [Lambda2](../Lambda2.png)
```

Comment:
In your README could you please link the tutorial that you used for this please!